examples/invocations/kritis-pipeline-run.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: pipeline.knative.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: kritis-pipeline-run-12321312984
+  namespace: default
+spec:
+    pipelineRef:
+        name: kritis-pipeline
+    pipelineParamsRef:
+        name: pipelineparams-sample
+    trigger:
+        triggerRef:
+            type: manual
+status:
+  taskRuns:
+    - taskRef:
+        name: unit-test-kritis-12321312984
+        nextTasks:
+          - taskRef:
+              name: push-kritis-12321312984
+        prevTasks: []
+    - taskRef:
+        name: push-kritis-12321312984
+        nextTasks: []
+        prevTasks:
+          - taskRef:
+            name: unit-test-kritis-12321312984
+  conditions:
```

Comment:
Add a comment here, this will be auto filed by the controller?

Reply:
ya the whole `status` section will be filled out by the controller - ill add a comment